tools/sdk-readme-table-generator/TableGenerator/Program.fs

```diff
@@ -12,6 +12,9 @@ let inputBranches =
           { GitBranchName = "release/9.0.1xx-preview2"
             DisplayName = "9.0.1xx-preview2<br>(9.0-preview2&nbsp;Runtime)"
             AkaMsChannel = Some("9.0.1xx-preview2/daily") }
```

Comment:
It should be preview3 here.

Reply:
@nagilson,
Please commit changes to Program.fs, too, or else I have to realize that you'd made changes without committing them and recreate them if I want to use the tool.

https://github.com/dotnet/installer/pull/19088

Reply:
Hm, yes I probably should have committed that change!